tests/test_custom_magma.py

```diff
@@ -0,0 +1,24 @@
+from peak import Peak, family_closure
+import magma as m
+from hwtypes import BitVector as BV
+import os
+
+def test_custom_verilog():
```

Comment:
what is this testing?

Reply:
The ability to specify a custom magma module in the __init__ instead of having it compile a peak class via magma.sequential2.

Reply:
Yes but what in peak are you testing?  If this a magma test it should be in magma.   Also this code will break if you invoke it with anything other than the magma family.  Remove this test or do it properly - make a new family where the magma family provides the wrapped verilog, the python family provides a functional model, etc...

Reply:
Removed the test